cfgov/jobmanager/models.py

```diff
@@ -102,8 +103,8 @@ def get_absolute_url(self):
 
     def save(self):
         if self.date_created == None:
```

Comment:
Fair point @vickumar1981! Although it's a grey area between picking up trash from the campground and mowing, weeding, planting some flowers... I've made the improvement you suggested and even added a couple of new tests.